containerized_ml_models/api/api.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from xgboost import XGBModel
+from flask import Flask, jsonify, request
+from containerized_ml_models.api import io
+
+MODEL_PATH: str = "containerized_ml_models/api/model.pkl"
+model: XGBModel = io.load_model(MODEL_PATH)
```

Comment:
Adnotacji typów zmiennych raczej się nie używa w komercyjnych projektach ale jak Ci pomagają to spoko.

Reply:
Nie jestem też pewny, czy robienie z modelu zmiennej globalnej jest dobrym pomysłem. Z jednej strony używasz go tylko w 1 endpoincie, z drugiej musiałbyś wczytywać za każdym razem. Chętnie usłyszę zdanie kogoś innego

Reply:
At this stage I think global variable is sufficent. However, it can be also done using singleton class encapsulating this model and exposing predict() method.

Reply:
Also could you provide more explanation about proper kind of typing?

Reply:
Not much of explanation, just variable: type = value isn't really used. It's time consuming and many variables have obvious type (like the string here)

Reply:
> At this stage I think global variable is sufficent. However, it can be also done using singleton class encapsulating this model and exposing predict() method.

It's more about should it sit here loaded in memory? What if some other method changes it? After all you only need it inside one endpoint. I do agree that it's a problem only when project becomes bigger

Reply:
I think that model should be loaded only once. In situation when we will have bigger model like hundreds of mb, then loading model will take much longer than prediction itself. In newest version I created singleton class which holds model internally. In this configuration it is unlikely that model will be accidentally changed.